airbyte-integrations/connectors/source-news-api/source_news_api/news_api.yaml

```
@@ -0,0 +1,68 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["articles"]
+  requester:
+    url_base: "https://newsapi.org/v2"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+    request_options_provider:
+      request_parameters:
+        q: "{{ config['search_query'] }}"
+        searchIn: "{{ ','.join(config['search_in']) }}"
+        sources: "{{ ','.join(config['sources']) }}"
+        domains: "{{ ','.join(config['domains']) }}"
+        excludeDomains: "{{ ','.join(config['exclude_domains']) }}"
```

This will break if `exclude_domains` is not defined in config.
```suggestion
        excludeDomains: "{{ ','.join(config['domains']) if config.get('domains') else None }}"
```

---

It actually works without, but I have made more robust by using `','.join(config.get('exclude_domains', []))`